modules/traf/files/buildtest/stage-traf.sh

```diff
@@ -113,11 +113,11 @@ cd $workspace
 
 # installer
 install=$(ls ./trafodion/install/installer*gz ./trafodion/distribution/installer*gz)
-cp $install ./$DestDir/installer-$BLD.tar.gz  || exit 2
+cp $install ./$DestDir/apache-trafodion-installer-$BLD-incubating-bin.tar.gz  || exit 2
```

Comment:
Instead of hardcoding (apache-trafodion) can we use the name from TRAFODION_VER_PROD environment variable